q2_diversity/_beta/_method.py

```diff
@@ -11,6 +11,7 @@
 import skbio
 import skbio.diversity
 import skbio.tree
+import sklearn.metrics
```

Comment:
I was looking everywhere for the requirements, haha! Haven't added a dependency since the CD refactors, so was really confused on why they weren't in the `setup.py` anymore.